tree/ntuple/v7/src/RNTupleSerialize.cxx

```diff
@@ -588,6 +588,7 @@ std::uint16_t ROOT::Experimental::Internal::RNTupleSerializer::SerializeFieldStr
          return SerializeUInt16(0x03, buffer);
       case ENTupleStructure::kReference:
          return SerializeUInt16(0x04, buffer);
+      case ENTupleStructure::kUnsplit: return SerializeUInt16(0x05, buffer);
```

Comment:
Let's make the formatting consistent with the other cases (which probably means the other cases need reformatting). The same holds for ll. 620.